apps/fluent-tester/src/FluentTester/testPages.macos.ts

```diff
@@ -33,11 +35,31 @@ export const tests: TestDescription[] = [
     component: ButtonTest,
     testPage: HOMEPAGE_BUTTON_BUTTON,
   },
+  {
+    name: 'Callout Test',
+    component: CalloutTest,
+    testPage: HOMEPAGE_CALLOUT_BUTTON,
+  },
+  {
+    name: 'ContextualMenu Test',
+    component: ContextualMenuTest,
+    testPage: HOMEPAGE_CONTEXTUALMENU_BUTTON,
+  },
   {
     name: 'Experimental Button',
     component: ExperimentalButtonTest,
     testPage: HOMEPAGE_BUTTON_BUTTONEXPERIMENTAL,
   },
+  {
+    name: 'Checkbox Test',
```

Comment:
Does it make sense to have the list be alphabetical so there's some organization to it?

Reply:
It's sorted before it's displayed, but fair

Reply:
fixed